### PR TITLE
[board] Add Nucleo-G070RB

### PR DESCRIPTION
**Pull request by chris-durand**
Add Nucleo-G070RB and blink example

### CODE REVIEW

File: src/modm/board/nucleo_g070rb/board.hpp [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+	static constexpr uint32_t Ucpd1		= Apb;
+	static constexpr uint32_t Dac		= Apb;
+	static constexpr uint32_t Pwr		= Apb;
+	static constexpr uint32_t I2c2		= Apb;
+	static constexpr uint32_t I2c1		= Apb;
+	static constexpr uint32_t Usart4	= Apb;
+	static constexpr uint32_t Usart3	= Apb;
+	static constexpr uint32_t Usart2	= Apb;
+	static constexpr uint32_t Spi2		= Apb;
+	static constexpr uint32_t Iwdg		= Apb;
```
Comment (salkinium) [inline src/modm/board/nucleo_g070rb/board.hpp:67]: I believe this should be `32_kHz`, but I'll fix in the IWDG PR.

**Review by salkinium — APPROVED** [review]
Yes, thanks!

File: src/modm/board/nucleo_g070rb/board.hpp
Comment (rleh) [inline src/modm/board/nucleo_g070rb/board.hpp:1]: Is/was it really necessary to add (duplicate) this file? [`src/modm/board/nucleo_g071rb/board.hpp`](https://github.com/chris-durand/modm/blob/027811f1d0b011236cb5d3c3d2a692032c821728/src/modm/board/nucleo_g071rb/board.hpp) has the same content, but adds a few peripherals in `SystemClock` struct that do not exist on the STM32G07**0** MCU.

Comment (salkinium) [reply src/modm/board/nucleo_g070rb/board.hpp:1]: Ah, I forgot about that idea, feel free to fix it. You then also need to add the g070 doxygen group to the file.

**Pull request merged**